core/src/navigation/TopNav.html

```diff
@@ -485,17 +485,20 @@
     } else {
       semicollapsedNav();
     }
-    if (NavigationHelpers.getBurgerTooltipConfig() !== undefined) {
-      setBurgerTooltip();
-    }
+    setBurgerTooltip();
   }
 
   function setBurgerTooltip() {
-    if (document.body.classList.contains('lui-simpleSlideInNav')) {
-      burgerTooltip = document.body.classList.contains('lui-leftNavToggle') ? NavigationHelpers.getBurgerTooltipConfig()[0] : NavigationHelpers.getBurgerTooltipConfig()[1];
-    }
-    if (document.body.classList.contains('lui-semiCollapsible')) {
-      burgerTooltip = document.body.classList.contains('semiCollapsed') ? NavigationHelpers.getBurgerTooltipConfig()[0] : NavigationHelpers.getBurgerTooltipConfig()[1];
+    if (NavigationHelpers.getBurgerTooltipConfig()) {
+      const [collapseNavTooltip, expandNavTooltip] = NavigationHelpers.getBurgerTooltipConfig();
+      if (collapseNavTooltip && expandNavTooltip) {
+        if (document.body.classList.contains('lui-simpleSlideInNav')) {
+          burgerTooltip = document.body.classList.contains('lui-leftNavToggle') ? collapseNavTooltip : expandNavTooltip;
```

Comment:
Same here, there are too many nested if condition. We should improve the readability.